tests/e2e_tests.py

```diff
@@ -33,6 +33,17 @@ def test_print_only_no_link(link_localtime_mock, deb_tz_mock):
     assert_false(deb_tz_mock.called)
 
 
+@httpretty.activate
+@mock.patch("tzupdate.write_debian_timezone")
+@mock.patch("tzupdate.link_localtime")
+def test_print_sys_tz_no_link(link_localtime_mock, deb_tz_mock):
+    setup_basic_api_response()
```

Comment:
The comment was marked as resolved, but again, why are we doing `setup_basic_api_response`? `--print-system-timezone` shouldn't need to reach out to get an API response

Reply:
Yeah, it was an oversight on my part, sorry